### PR TITLE
3.6.0

### PR DESCRIPTION
**Pull request by richardwestenra**
# Release 3.6.0

## Major features and improvements

- Redesign main sidebar (#236, #283) 
- Drop Kedro 0.14.* support (#277)

## Bug fixes and other changes

- Continue work-in-progress on the multiple pipeline selection dropdown, which is still hidden behind a flag and disabled by default but is nearly complete. (#270, #273, #285, #289)
- Continue work on new metadata panel endpoints (#275)
- Fix chart rendering edge cases and hover styles (#288)
- Update Python unit tests using the same json file as front-end (#281)
- Improve lib-test docs (#278)
- Hide random seed message unless using random data (#280)
- Delete deprecated isParam and schema_id fields (#274)
- Fix bug caused by typo in saveStateToLocalStorage (#271)
- Fix interrupted chart transitions (#269)
- Refactor and optimise flowchart performance (#268)

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -32,7 +52,8 @@ Please follow the established format:
 - Add web worker to make the expensive graph layout calculation into an asynchronous action, to prevent it from blocking other tasks on the main thread. (#217)
 - Focus search bar with Cmd+F/Ctrl+F keyboard shortcuts (#261)
 - Allow an argument to be passed to loadJsonData, for external use if needed (#215)
-- Add support for multiple pipelines. This is a work-in-progress, and is currently disabled by default and hidden behind a flag. (#192, #215, #216, #221, #252, #254)
+- Add support for multiple pipelines. This is a work-in-progress, and is currently disabled by default and hidden behind a flag. (#192, #215, #216, #221, #254)
+- Begin adding individual node API endpoints, as a prelinary step towards full node metadata sidebars (#252)
```
Comment (921kiyo) [inline RELEASE.md:56]: Was this missing from the last release?

**Review by 921kiyo — APPROVED** [review]
💯

File: RELEASE.md (hunk 1 above)
Comment (richardwestenra) [reply RELEASE.md:56]: I had included `#252` under the wrong bullet point, so I've fixed it. It wasn't really part of the multiple pipelines feature, it was part of the metadata panel feature

**Review by limdauto — APPROVED** [review]

**Review by studioswong — APPROVED** [review]

**Review by bru5 — APPROVED** [review]
